json/json-derivation/src/main/scala/io/sphere/json/generic/package.fmpp.scala

```diff
@@ -24,6 +24,7 @@ package object generic extends Logging {
   type JSONIgnore = io.sphere.json.annotations.JSONIgnore @getter
   type JSONTypeHint = io.sphere.json.annotations.JSONTypeHint
   type JSONTypeHintField = io.sphere.json.annotations.JSONTypeHintField
+  type JSONCodec[T] = FromJSON[T] with ToJSON[T]
```

Comment:
Could we use the existing `JSON` type instead? https://github.com/commercetools/sphere-scala-libs/blob/99c9fb0c96c758135f2e91ece7b14a809dc6ee29/json/json-core/src/main/scala/io/sphere/json/JSON.scala#L8